### PR TITLE
Add Big Chunk of Code

### PR DESCRIPTION
**Pull request by initseis**
In this branch, I added my Code from my To-Do-List project.

### CODE REVIEW

**Comment by leonardodiasb** [comment]
Hi @sevinchek , 

Good job so far! 🔥🔥 The first step for the to-do list it's taken it, your app has a style and the basic setup it's ready.

There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

this close

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/drag.js [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+            checkDiv.appendChild(descTextArea);
+            const dragBtn = document.createElement('button');
+            dragBtn.draggable = 'true';
+            const imgDots = document.createElement('img');
+            imgDots.src = 'img/three-dots.svg';
+            imgDots.alt = '3 dots icon';
+            imgDots.width = '15';
+            dragBtn.appendChild(imgDots);
+            toDoLi.appendChild(dragBtn);
+          }
```
Comment (leonardodiasb) [inline src/drag.js]: -Maybe you could simplify this code by putting it out the if statement and only use the statement to apply the `checks.checked = 'true';` code to it.

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    liElements[i].addEventListener('drop', drag.drop);
+  }
+  const itemDesc = document.querySelectorAll('.item-description');
+  for (let j = 0; j < liElements.length; j += 1) {
+    const crud = new Crud();
+    itemDesc[j].addEventListener('input', crud.updateTask); // Update Elements
+    itemDesc[j].addEventListener('focusin', crud.showTrashAndRemove); // Remove Elements
+    itemDesc[j].addEventListener('focusout', crud.showDots);
+  }
+}
```
Comment (leonardodiasb) [inline src/index.js]: Try to don't use the old for loop in all your JS files

File: src/status.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
+export default class Status {
+  validation() {
+    if (this.checked === true) {
+      this.nextSibling.style['text-decoration'] = 'line-through';
+      this.nextSibling.style.color = '#909090';
+      // Save in Local Storage
+      Status.saveChanges();
```
Comment (leonardodiasb) [inline src/status.js]: Use conditional shorthands. `(this.checked)`, `(this.checked === true)` is not necessary.

File: src/drag.js (hunk 1 above)
Comment (initseis) [reply src/drag.js]: You are right. 💯

File: src/index.js (hunk 2 above)
Comment (initseis) [reply src/index.js]: I did it. 💪🏽

File: src/status.js (hunk 3 above)
Comment (initseis) [reply src/status.js]: Sure. Thanks. 💯

**Comment by initseis** [comment]
Thanks for the review, @leonardodiasb. I did the changes you suggested to me. 🔥🔥

**Comment by leonardodiasb** [comment]
Hi @sevinchek , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**